aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -741,67 +711,110 @@ The following example permits HTTP/1.1 and HTTP/2 connections on port 8000. Conn
 });
 ```
 
-Optionally use Connection Middleware to filter TLS handshakes on a per-connection basis for specific ciphers:
+Use Connection Middleware to filter TLS handshakes on a per-connection basis for specific ciphers if required.
+
+The following example throws <xref:System.NotSupportedException> for any cipher algorithm that the app doesn't support. Alternatively, define and compare [ITlsHandshakeFeature.CipherAlgorithm](xref:Microsoft.AspNetCore.Connections.Features.ITlsHandshakeFeature.CipherAlgorithm) to a list of acceptable cipher suites.
+
+[CipherAlgorithmType.Null](xref:System.Security.Authentication.CipherAlgorithmType) indicates that no cipher algorithm supported by Kestrel matches the requested algorithm.
```

Comment:
No, Null is the name of an algorithm that can be negotiated (a very bad one).
https://docs.microsoft.com/en-us/dotnet/api/system.security.authentication.cipheralgorithmtype?view=netframework-4.8

Reply:
Thanks ... I'll fix that.